clusterloader2/testing/dra/job.yaml

```
@@ -23,8 +23,15 @@ spec:
           args:
             - {{.Sleep}}
           resources:
+          {{ if not .ExtendedResource }}
             claims:
             - name: gpu
+          {{ else }}
+            limits:
+              example.com/gpu: "1"
+          {{ end }}
+      {{ if not .ExtendedResource }}
       resourceClaims:
       - name: gpu
-        resourceClaimTemplateName: single-gpu-0
\ No newline at end of file
+        resourceClaimTemplateName: single-gpu-0
+      {{ end }}
```

```suggestion
          {{ if .ExtendedResource }}
            limits:
              example.com/gpu: "1"
          {{ else }}
            claims:
            - name: gpu
      resourceClaims:
      - name: gpu
        resourceClaimTemplateName: single-gpu-0
      {{ end }}
```
Prefer positive logic

---

fixed